src/main/resources/swagger.api/sharing_instance.yaml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.0
+info:
+  title: Sharing instance integration API
+  description: "Sharing instance integration API"
+  version: 0.0.1
+servers:
+  - url: /consortia/{consortiumId}/
+paths:
+  /sharing/instances:
+    post:
+      summary: Save Sharing instance
```

Comment:
The same regarding renaming, Start instance sharing

Reply:
Changed